src/TheCallCenter/Startup.cs

```diff
@@ -35,11 +31,11 @@ public void ConfigureServices(IServiceCollection services)
       });
 
 
-      services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+      services.AddMvc();
```

Comment:
services.AddControllers();

Reply:
If you look at the latest changes from last week, you'll see I already resolved that. 

This is what it says now.

      /* 
       * 2023-06-12 - Differ from Video: 
       * Replaces AddMvc() with AddControllersWithViews(). 
       * AddMvc Enables Razor Pages when we don't need that feature. Adding
       * Mvc instead of just "ControllersWithViews" adds a performance hit that's
       * unnecessary
       */
      services.AddControllersWithViews();

That's why I resolved the issue. Please look at my latest version of pull request.